engine/classes/Elgg/Database/EntityTable.php

```diff
@@ -1319,17 +1317,12 @@ public function getEntityAttributeWhereSql(array $options = array()) {
 	 * @param string $type           The type of entity
 	 * @param string $subtype        The subtype of entity
 	 * @param int    $container_guid The container GUID that the entities belong to
-	 * @param int    $site_guid      The site GUID
+	 * @param int    $ignored        Ignored function parameter
 	 * @param string $order_by       Order_by SQL order by clause
 	 *
 	 * @return array|false Either an array months as YYYYMM, or false on failure
 	 */
-	public function getDates($type = '', $subtype = '', $container_guid = 0, $site_guid = 0, $order_by = 'time_created') {
-
-		$site_guid = (int) $site_guid;
-		if ($site_guid == 0) {
-			$site_guid = $this->config->get('site_guid');
-		}
+	public function getDates($type = '', $subtype = '', $container_guid = 0, $ignored = 0, $order_by = 'time_created') {
```

Comment:
Private API, remove the argument, or better yet update to accept an array of options